### PR TITLE
feat: complete training section

### PR DESCRIPTION
**Pull request by Samet91**

### CODE REVIEW

File: style/components/training.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
+.text_positioning {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  
+}
+
+.img_subtitle {
```
Comment (F-Kirchhoff) [inline style/components/training.css:9]: .training__img-subtitle

File: style/components/training.css [hunk 2]
```diff
@@ -0,0 +1,22 @@
+.text_positioning {
```
Comment (F-Kirchhoff) [inline style/components/training.css:1]: .training__test-postitioning

File: style/components/training.css [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  display: flex;
+  position: absolute;
+  color: var(--c-white);
+  font-size: 1.5rem;
+  font-weight: 500;
+  left: 7px;
+  bottom: 30px;
+}
+
+.training-img {
```
Comment (F-Kirchhoff) [inline style/components/training.css:19]: training__img

File: style/components/workout-button.css [hunk 4]
```diff
@@ -0,0 +1,7 @@
+.button {
```
Comment (F-Kirchhoff) [inline style/components/workout-button.css:1]: workout-button

File: style/components/section-title.css [hunk 5]
```diff
@@ -0,0 +1,12 @@
+.title{
```
Comment (F-Kirchhoff) [inline style/components/section-title.css:1]: section-title

File: index.html [hunk 6]
```diff
@@ -1,63 +1,77 @@
...
-
-      <div class="navbar__button-container">
-        <button class="navbar__Login">Login</button>
-        <button class="navbar__Sign-in">Sign in</button>
+  <body>
+    <nav class="navbar">
+      <div class="logo">
+        <p class="logo__text">
+          Work<span class="logo__text logo__text--pink">out</span>
+        </p>
```
Comment (WaliedWeb) [inline index.html:16]: good idea

**Pull request merged**